### PR TITLE
feat: added Home, Header and Footer components

### PR DESCRIPTION
**Pull request by md-amir-gauhar**
Screenshot (193)
Screenshot (191)
Screenshot (192)

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="footer-top">
+        <section className="about">
+          <span>About</span>
+          <a href="#">Contact Us</a>
+          <a href="#">About Us</a>
+          <a href="#">Press</a>
```
Comment (samflab) [inline src/components/Footer.jsx]: don't forget to change these to `Link` tag

**Review by samflab — APPROVED** [review]
looks good.
there are some files in which you have used `px` , instead use `rem` or `em` for uniformilty.

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          <a href="#">Shipping</a>
+          <a href="#">Cancellation & return</a>
+          <a href="#">FAQ</a>
+        </section>
+        <section className="social">
+          <span>Social</span>
+          <a href="#"><ImTwitter style={{ color: "#1A8CD8" }} /></a>
+          <a href="#"><BsFacebook style={{ color: '#fff' }} /></a>
+          <a href="#"><BsLinkedin style={{ color: '#0077B5' }} /></a>
+        </section>
```
Comment (manismk) [inline src/components/Footer.jsx]: try to avoid using inline styles

**Review by manismk — APPROVED** [review]
Looks good. clean UI

**Pull request merged**